README.md

```diff
@@ -1,35 +1,80 @@
+[Orion Context Broker](https://github.com/telefonicaid/fiware-orion).
 
--   [Architecture](documentation/architecture.md)
--   [Logs & Alarms](documentation/logs.md)
--   O&M
+This project is part of [FIWARE](https://www.fiware.org). You can find more FIWARE components in the
+[FIWARE catalogue](https://catalogue.fiware.org).
 
-    -   [Deployment](documentation/deployment.md)
-    -   [Configuration](documentation/configuration.md)
-    -   [Administration](documentation/admin.md)
-    -   [Metrics API](documentation/metrics_api.md)
+| :books: [Documentation](https://perseo.readthedocs.io/en/latest) | :whale: [Docker Hub](https://hub.docker.com/r/fiware/perseo/) | :dart: [Roadmap](documentation/roadmap.md) |
+| ---------------------------------------------------------------- | ------------------------------------------------------------- | ------------------------------------------ |
 
--   [API](documentation/api.md)
-    -   [Plain rules](documentation/plain_rules.md)
-    -   [Errors](documentation/errors.md)
--   [Data model](documentation/models.md)
--   [Available actions for PEP](documentation/pep_actions.md)
--   [Documentation for developers of perseo](documentation/development.md)
 
-| :dart: [Roadmap](documentation/roadmap.md) |
-| ------------------------------------------ |
+## Content
 
+-   [Background](#background)
+-   [Installation](#installation)
+-   [Usage](#usage)
+-   [API](#api)
+-   [Testing](#testing)
+-   [Quality Assurance](#quality-assurance)
+-   [More Information](#more-information)
+-   [License](#license)
+
+## Background
+
+Perseo is an [Esper](http://www.espertech.com/esper/)-based Complex Event Processing (CEP) software supporting the
+NGSIv2 protocol.
+
+It follows a straightforward idea: listening to events coming from context information to identify patterns described by
+rules, in order to immediately react upon them by triggering actions.
+
+![Perseo Components](docs/images/PerseoComponents.png)
+
+By leveraging on the
+[notifications mechanism](http://fiware-orion.readthedocs.io/en/latest/user/walkthrough_apiv2/index.html#subscriptions),
+clients instruct Orion CB to notify Perseo of the changes in the entities they care about (`Event API`). Details of this
+process are explained in the [Orion Subscription part of the User Manual](user/index.md#orion-subscription). Then, rules
+to the CORE Rule Engine can be easily managed using the publicly available WireCloud operational dashboard, or making
+use of any of the REST clients able to programmaticly use the Perseo's `Rule API`. These rules will identify patterns
+that will trigger actions with Orion to create or update entities, or with other different components or external
+systems, such as Web (HTTP), Email (SMTP) or SMS (SMPP) servers.
+
+## Installation
+
+The instructions to install Perseo can be found in the
+[Installation Guide](https://perseo.readthedocs.io/en/latest/setup.html)
```

Comment:
Whenever it possible use the markdown-to-markdown link instead of markdown-to-RTD link. Have a look to the Orion README.md for instance (https://github.com/telefonicaid/fiware-orion/blob/master/README.md).

(Not only this one but also the other cases in this file)

Reply:
Fixed in dc133ec86b6827a04700dfe498b3516dedf4e19c